oca_custom/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    github_login = fields.Char(readonly=False)
```

Comment:
This field was changed in the refactor to v14. Now it is called `github_name`. IMHO, that makes more sense, as it is a field shared across all github related models, and is not always a “login”. I have done migration scripts in the refactor, so you would just need to change this to:
```suggestion
    github_name = fields.Char(readonly=False)
```